turbo/generators/templates/appPackage/sample-manifest.json.hbs

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.20/MicrosoftTeams.schema.json",
+  "version": "1.0.0",
+  "manifestVersion": "1.20",
+  "id": "$\{{TEAMS_APP_ID}}",
+  "name": {
+    "short": "{{ name }}-test-sample",
+    "full": "{{ name }}-test-sample"
```

Comment:
Can we go with `template` instead of `test-sample`?

Reply:
i thought we didn't wanna call it templates haha

Reply:
you're right, myb, I left the comment before I asked about the hbs files, sorry!